openmc/lattice.py

```diff
@@ -93,9 +93,8 @@ def universes(self):
     @id.setter
     def id(self, lattice_id):
         if lattice_id is None:
-            global AUTO_UNIVERSE_ID
```

Comment:
nvm, I see why it is not needed here since it is imported from a different file unlike the other cases